framework/auth/core.py

```diff
@@ -848,6 +854,19 @@ def _get_unconfirmed_email_for_token(self, token):
 
         return verification['email']
 
+    def clean_email_verifications(self, **kwargs):
+        given_token = kwargs.get('given_token')
```

Comment:
I am using it in two contexts in views.py.  In unconfirmed_email_remove I am removing a specific one, in auth_login I am removing all the dead weight of expired tokens so it won't continue to poll for expired tokens on every page load.  Can you think of a better way to architect it?